### ISSUE TITLE
Errors 

### ISSUE THREAD

**Issue by Ashutoshk1** [blockchain/blockchain-wallet-v4-frontend#3144]
Hey , can anyone solve my issue too.I wrote my issue many time to support .But no one is responding.When I contacted through live chat,they simply say my password is wrong and can't be recovered.

My issue is whenever I enter my password and try to login , I see a message error in decryption and password wrong.

But my password is correct.The day before I was able to login.I saved my password to Google password manager.So,I am 100% sure my password is correct.I am sure this is technical issue which started probably since the day the website version was updated.Now, can anyone please fix this issue.

I also find on reddit many people facing the same issue.But the solution which they are providing is for engineers.I can't understand .

I am submitting the link below.Pleqse check the first link.They have provided some solution.But I don't understand jdk and all.So, please fix this.

blockchain/service-my-wallet-v3#205

https://bitcointalk.org/index.php?topic=3313785.0

https://amp.reddit.com/r/Bitcoin/comments/jybfxq/wallet_decryption_error_with_correct_password/

Also let me delete my comment on previous thread due to privacy reasons. 

Thank you

**Comment by Ashutoshk1** [comment]
Apart from this , I also not receiving password reset link on my mail for exchange account.

However,  no need to reset as I know password but when I try to login , message appears invalid credentials

**Comment by Ashutoshk1** [comment]
If don't know to help.Its ok but not tell a lie that I forgot password. 

Now I want to delete my thread. How can I do that?
I suppose atleast this you can help

**Comment by schnogz** [comment]
You cant delete issues created on Github. The internet doesnt forget things my friend.

**Comment by schnogz** [comment]
So you have gone through all this trouble, making support tickets with our company, multiple github issues all for one dollar? 
 And now you also want to delete your thread? 
 
 Are you worried that people will figure out you're probably trying to scam?

**Comment by Ashutoshk1** [comment]
Yes, I have gone through all those poor support services by blockchain.And multiple github issues got created due to you people are just closing the thread without reading my issues properly and misinterpreting also that I forgot the password and I am trying to scam. If I forgot the password than how I am seeing this message. These screen  appears only appears after entering 12 words phrase or password along with gmail.

![Screenshot_20210430-184529_Gallery](https://user-images.githubusercontent.com/83293968/116753669-105d6480-aa25-11eb-99dd-53dcb80e1cfd.jpg)
![Screenshot_20210429-092753_Gallery](https://user-images.githubusercontent.com/83293968/116753705-2408cb00-aa25-11eb-891d-5ed5a9d737fc.jpg)


If you read carefully. You will understand the issues .And regarding one dollar.Yes, it is just less than 1$ in blockchain exchange.Than you might ask why I am eager to access that account.Because for me even that much amount matter because I worked many days on 2 captcha to earn that much.And second purpose is I am testing blockchain support services.So, that I can review.

Since 3 months and more even on chat support service, you people just says to wait 24 hours to check mail.But no response after waiting.Likewise, you people just waste our time.

Anyway, no issue I can use some other crypto websites.
AND REGARDING MY OWNERSHIP,  DO YOU NEED MY PASSWORD OF GMAIL TO CONSIDER THAT IT IS MY ACCOUNT.

Hope you understands...
